jobs/scripts/parallel-tests.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+sed -i 's/    mr.developer/    mr.developer\ngit-clone-depth = 10/' core.cfg
+
+pip install -r requirements.txt
+buildout -c py3.cfg
```

Comment:
We can add `buildout:git-clone-depth=1` here. The syntax is `section:option=value`.

`buildout buildout:git-clone-depth=1 -c py3.cfg`